dags.yaml

```diff
@@ -838,3 +838,23 @@ bqetl_telemetry_crashes_daily:
   tags:
     - impact/tier_2
     - repo/bigquery-etl
+
+bqetl_internal_tooling_wchan:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - wichan@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
```

Comment:
```suggestion
```

I think you don't need to specify this unless there is an end_date to the DAG's scheduling.